cypress/integration/examples/querying.spec.js

```diff
@@ -0,0 +1,114 @@
+/// <reference types="cypress" />
```

Comment:
Change the file name to something more descriptive and relevant to the task, like 'book-store.spec.js'.